source/velocity_boundary_conditions/gplates.cc

```diff
@@ -741,19 +590,60 @@ namespace aspect
                 ExcMessage ("To define a plane for the 2D model the two assigned points "
                             "may not be equal."));
 
-      lookup.reset(new internal::GPlatesLookup(pointone,pointtwo,interpolation_width));
-
-      const GeometryModel::Interface<dim> &geometry_model =
-        this->get_geometry_model();
-      (void)geometry_model;
-      Assert (dynamic_cast<const GeometryModel::SphericalShell<dim>*> (&geometry_model) != 0,
-              ExcMessage ("This boundary condition can only be used if the geometry "
-                          "is a spherical shell."));
-      Assert((dynamic_cast<const GeometryModel::SphericalShell<dim>*> (&geometry_model))->opening_angle()==360,
-             ExcMessage("The gplates velocity model works only for an opening "
-                        "angle of the geometry equal to 360."));
-    }
+      if ((dynamic_cast<const GeometryModel::SphericalShell<dim>*> (&this->get_geometry_model())) != 0)
+        {
+          lookup.reset(new internal::GPlatesLookup<dim>(pointone,pointtwo));
+          old_lookup.reset(new internal::GPlatesLookup<dim>(pointone,pointtwo));
+        }
+      else
+        AssertThrow (false,ExcMessage ("This gplates plugin can only be used when using "
+                                       "a spherical shell or box geometry."));
```

Comment:
We do not allow box geometries any longer. But instead you should add a ` || (dynamic_cast ... Chunk<dim> ... != 0)` to the if statement above. The spherical chunk should work with no problems with the new gplates plugin.